src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -982,3 +990,34 @@ def _get_trigger_event_list_patch(cmd,
             if SourceTriggerEvent.pullrequest.value in source_trigger_events:
                 source_trigger_events.remove(SourceTriggerEvent.pullrequest.value)
     return source_trigger_events
+
+def prepare_context_path(source_location, client_registries, registry_name, resource_group_name):
```

Comment:
do we already have similar function?

Reply:
yes this is pretty much the same as prepare_source_location in run.py

Reply:
@ElianaLiberman  would you mind moving this function to the utils so we can have the same code running for both task and run?